src/io/flutter/actions/OpenInAndroidStudioAction.java

```diff
@@ -151,6 +151,9 @@ private static void openFileInStudio(@NotNull VirtualFile projectFile, @NotNull
       }
       else {
         // TODO Open editor on sourceFile for Linux, Windows
```

Comment:
Should we adjust this TODO? I think it describes what this method is doing; we can perhaps delete the todo.

Reply:
The TODO is still valid. I plan to experiment to see if we can get a file to open, like with MacOS.